R/deps.R

```diff
@@ -16,6 +16,14 @@
 #'   "Suggests". `NA` is shorthand for "Depends", "Imports" and "LinkingTo"
 #'   and is the default. `FALSE` is shorthand for no dependencies (i.e.
 #'   just check this package, not its dependencies).
+#'
+#'   The values "soft" or "dev" mean the same as `TRUE`, "hard" or "user" mean
+#'   the same as `NA`.
+#'
+#'   You can also specify dependencies from one or more additional fields,
+#'   common ones are "Config/pkgdown" for dependencies used in building the
```

Comment:
I wonder if we should use a more generic name, like "website"`? I'd also make this a bulleted list now, since we're likely to slowly add other fields.

Reply:
Do we want to add `Config/Remote/development` now? (i.e. you'd put roxygen2 in this field)

Reply:
Was thinking of starting slow, with just things we need on CI systems and go from there...